Automattic/pocket-casts-android
# Hack Week Project: Add options to sort Up Next

## Description
- This is part of the Hack Week project, which adds 3 options to sort Up next
- ⚠️ If this gets approved, we will need to review the design proposed here
- ⚠️ To merge this PR, we need to do some regression tests first
- See: pdjfGa-6h-p2


## Testing Instructions
- Add some episodes to up next and see the sort icon at the top of the screen
- Try all options to sort up next

## Screenshots or Screencast 

[Screen_recording_20241203_144705.webm](https://github.com/user-attachments/assets/54e7b2d4-7f3b-4e11-a908-95344f47577b)

| Up Next | Modal |
|--------|--------|
| ![Screenshot_20241203_144630](https://github.com/user-attachments/assets/4feb9900-4890-43b6-853d-43aef3d61fd0) | ![Screenshot_20241203_144642](https://github.com/user-attachments/assets/dd21d8c5-99e9-40a6-afe2-21454e8ca652)
 |
